pyblazing/pyblazing/apiv2/context.py

```diff
@@ -711,12 +750,95 @@ def localfs(self, prefix, **kwargs):
 
     # Use result, error_msg = hdfs(args) where result can be True|False
     def hdfs(self, prefix, **kwargs):
+        """
+        Register a Hadoop Distributed File System (HDFS) Cluster.
+        
+        Parameters
+        ----------
+        
+        name : string that represents the name with which you will refer to your HDFS cluster.
+        host : string IP Address of your HDFS NameNode.
+        port : integer of the Port number of your HDFS NameNode.
+        user : string of the HDFS User on your NameNode.
+        kerb_ticket (optional) : string file path to your ticket for kerberos authentication.
+        
```

Comment:
Please mention that the user needs to setup environment variables for HDFS to work and list them

Reply:
https://github.com/gumdropsteve/blazingsql/commit/948e0f8f5a0dc684715830af12c2356286327f7f#diff-b733e955c1d24a9bd95af9c95b18d70fR765-R770